src/assets/curriculum/moving-straight-ahead/moving-straight-ahead.json

```diff
@@ -4,6 +4,33 @@
   "title": "Moving Straight Ahead",
   "subtitle": "Linear Expressions, Equations, and Relationships",
   "placeholderText": "Enter notes here",
+  "sections": {
+    "introduction": {
+      "initials": "In",
+      "title": "Introduction",
+      "placeholder": "Work area for\nIntroduction section"
+    },
+    "initialChallenge": {
+      "initials": "IC",
+      "title": "Initial Challenge",
+      "placeholder": "Work area for\nInitial Challenge section"
+    },
+    "whatIf": {
+      "initials": "W?",
```

Comment:
WI and NW were used in the mock-up.  I don't think it matters for now.

Reply:
I'll just add a commit to switch to those abbreviations in the interest of avoiding any further conversation.